### DISCUSSION TITLE
Can VM Snapshot be used for restoring ETCD cluster?

### DISCUSSION THREAD

**Discussion by pranay142** [etcd-io/etcd#18554] [Q&A] [answered]
We have a 3 node etcd cluster running on a VM and would like to know if the VM Snapshot can be used for restoring etcd in case of disaster recovery.
Kindly suggest.

**Comment by ahrtr** [comment, marked as answer]
It should work. It's easy to restore the etcd cluster as long as you have an etcd backup, which can be either a snapshot created by `etcdctl snapshot save` or a db file (from ${DATA_DIR}/member/snap/db). If you have a VM snapshot, so I assume you have an etcd db file at least.
